src/main/java/ch/njol/skript/expressions/ExprEntities.java

```diff
@@ -146,84 +145,80 @@ public boolean isLoopOf(String s) {
 	@Override
 	@Nullable
```

Comment:
while you're here, this should be fixed to be an `Entity @Nullable []` annotation instead